dot/parachain/backing/candidate_backing.go

```diff
@@ -1,121 +1,664 @@
 	}
+	return nil
 }
 
 func (cb *CandidateBacking) handleActiveLeavesUpdate() {
 	// TODO: Implement this #3503
 }
 
-func (cb *CandidateBacking) handleGetBackedCandidates() {
+func (cb *CandidateBacking) handleGetBackedCandidatesMessage() {
 	// TODO: Implement this #3504
 }
 
-func (cb *CandidateBacking) handleCanSecond() {
+func (cb *CandidateBacking) handleCanSecondMessage() {
 	// TODO: Implement this #3505
 }
 
-func (cb *CandidateBacking) handleSecond() {
+func (cb *CandidateBacking) handleSecondMessage() {
 	// TODO: Implement this #3506
 }
 
-func (cb *CandidateBacking) handleStatement() {
-	// TODO: Implement this #3507
+// Import the statement and kick off validation work if it is a part of our assignment.
+func (cb *CandidateBacking) handleStatementMessage(
+	relayParent common.Hash,
+	signedStatementWithPVD SignedFullStatementWithPVD,
+	chRelayParentAndCommand chan RelayParentAndCommand,
+) error {
+	rpState, ok := cb.perRelayParent[relayParent]
+	if !ok {
+		return fmt.Errorf("Received statement for unknown relay parent %s", relayParent)
+	}
+
+	summery, err := rpState.importStatement(cb.SubSystemToOverseer, signedStatementWithPVD, cb.perCandidate)
+	if err != nil {
+		return fmt.Errorf("importing statement: %w", err)
+	}
+
+	if err := postImportStatement(); err != nil {
+		return fmt.Errorf("post import statement: %w", err)
+	}
+
+	if summery == nil || summery.GroupID != uint32(rpState.Assignment) {
+		return nil
+	}
+
+	statementVDT, err := signedStatementWithPVD.SignedFullStatement.Payload.Value()
+	if err != nil {
+		return fmt.Errorf("getting value from statementVDT: %w", err)
+	}
+
+	var attesting AttestingData
+	switch statementVDT.Index() {
+	case 1: // Seconded
+		commitedCandidateReceipt, err := rpState.Table.getCandidate(summery.Candidate)
+		if err != nil {
+			return fmt.Errorf("getting candidate: %w", err)
+		}
+
+		candidateReceipt := parachaintypes.CandidateReceipt{
+			Descriptor:      commitedCandidateReceipt.Descriptor,
+			CommitmentsHash: common.MustBlake2bHash(scale.MustMarshal(commitedCandidateReceipt.Commitments)),
+		}
+
+		attesting = AttestingData{
+			candidate:     candidateReceipt,
+			povHash:       statementVDT.(parachaintypes.Seconded).Descriptor.PovHash,
+			fromValidator: signedStatementWithPVD.SignedFullStatement.ValidatorIndex,
+			backing:       []parachaintypes.ValidatorIndex{},
+		}
+
+		rpState.fallbacks[summery.Candidate] = attesting
+
+	case 2: // Valid
+		attesting, ok = rpState.fallbacks[summery.Candidate]
+		if !ok {
+			return nil
+		}
+
+		ourIndex := rpState.TableContext.validator.index
+		if signedStatementWithPVD.SignedFullStatement.ValidatorIndex == ourIndex {
+			return nil
+		}
+
+		if rpState.AwaitingValidation[summery.Candidate] {
+			// Job already running
+			attesting.backing = append(attesting.backing, signedStatementWithPVD.SignedFullStatement.ValidatorIndex)
+			return nil
+		}
+		// No job, so start another with current validator:
```

Comment:
It would be useful to log `no job, so start another with current validator` in debug mode

It this leads to too many logs, we can always remove it later